cli/src/commands/extensions/mod.rs

```diff
@@ -0,0 +1,132 @@
+pub mod extension;
```

Comment:
The combination of `pub mod extension;` here and `pub use extension::*;` below means that there are 2 valid public paths for everything in that module (i.e., `use crate::commands::extensions::extension::Extension;` or `use crate::commands::extensions::Extension;`). It's cleaner if at most one of these statements is `pub`.

```suggestion
mod extension;
```